test/e2e/app-dir/parallel-routes-and-interception/next.config.js

```diff
@@ -2,6 +2,9 @@
  * @type {import('next').NextConfig}
  */
 const nextConfig = {
+  experimental: {
+    ppr: process.env.__NEXT_EXPERIMENTAL_PPR === 'true',
+  },
```

Comment:
This shouldn't be required, that's the applied default already.

Reply:
Only in CI. It's useful when running `__NEXT_EXPERIMENTAL_PPR=true pnpm next build test/e2e/app-dir/parallel-routes-and-interception` locally though. We already have that in a bunch of test configs for this reason.

Reply:
Running this locally:

```bash
__NEXT_EXPERIMENTAL_PPR=true pnpm next build test/e2e/app-dir/parallel-routes-and-interception
```

Will also work with those lines removed from the `next.config.js`.

Reply:
It does not. You would need to add `__NEXT_TEST_MODE` as well, but that also changes a few other things, which I'd rather not have happening.